microservice-speaker/microservice-speaker-web/src/main/java/io/microprofile/showcase/speaker/rest/Resource.java

```diff
@@ -19,32 +19,71 @@
 import io.microprofile.showcase.speaker.model.Speaker;
 import io.microprofile.showcase.speaker.persistence.SpeakerDAO;
 
+import javax.annotation.PostConstruct;
 import javax.ejb.Lock;
 import javax.ejb.LockType;
 import javax.ejb.Singleton;
+import javax.ejb.Startup;
+import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
+import javax.ws.rs.DELETE;
 import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.PUT;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
-import java.util.HashSet;
+import java.util.Collections;
 import java.util.Set;
 
 /**
  * The Speaker resource
  */
 @Singleton
+@Startup
 @Lock(LockType.READ)
+@Produces(MediaType.APPLICATION_JSON)
 @Path("/")
 public class Resource {
```

Comment:
Is Resource the right name for this class?  We're dealing with processing Speaker related requests